src/data/community-events.json

```diff
@@ -589,9 +589,9 @@
     "to": "https://ethsafari.xyz/",
     "sponsor": null,
     "location": "Nairobi & Kilifi, Kenya",
-    "description": "Welcome to the latest greatest ETH event happening in Africa!",
-    "startDate": "2022-09-18",
-    "endDate": "2022-09-25"
+    "description": "Welcome to the latest greatest ETH event happening in Africa! Join the BlockTrain from Nairobi to celebrate an ETH-festival held underneath ancient Boabab trees in Kilifi.",
```

Comment:
BlockTrain? Is this a typo @Barukimang?

Reply:
Hi @minimalsm No it's an actual train which attendees can take from Nairobi to Kilifi. We call it the BlockTrain for the occasion :) On the train ppl can hack and just meet and bond with other attendees. It's a nice ride through national parks with lots of elephants

Reply:
Sounds wonderful. Thanks for the quick reply.